src/subscription_manager/utils.py

```diff
@@ -617,3 +617,18 @@ def unique_list_items(l, hash_function=lambda x: x):
 
 def generate_correlation_id():
     return str(uuid.uuid4()).replace('-', '')  # FIXME cp should accept -
+
+
+def get_process_names():
+    for subdir in os.listdir('/proc'):
+        if re.match('[0-9]+', subdir):
+            with open(os.path.join(os.path.sep, 'proc', subdir, 'status')) as status:
+                name = status.readline().split('\t')[-1].strip()
```

Comment:
The API of `/proc` is quite stable, but some exotic platform could be a little bit different and then we would have have a problem. So I would try to be more robust and more descriptive here. I would use something like this:

```python
process_status_file_path = os.path.join(os.path.sep, 'proc', subdir, 'status')
with open(process_status_file_path) as status_file:
    line = status_file.readline()
    if len(line) > 0:
        line = line.split('\t')
        if len(line) >= 2:
            if line[0] == 'Name:':
                proc_name = line[1].strip()
                yeild proc_name       
```
It will be necessary to e.g. raise some exception, when the content of the status file contains empty line, does not contain 'Name:' on the first line, etc. I would also add comment with typical content of `/proc/<proc_id>/status file. It is also important explain in comment what we are trying to do here and why try to do it in this way (why we do not use e.g. `psutil` Python package).